pkg/builder.go

```diff
@@ -28,7 +27,7 @@ const (
 // Git.
 //
 // Run returns on of the Status* status code constants.
-func Run(sshHostIP string, sshHostPort int, cmd string) int {
+func RunBuilder(sshHostIP string, sshHostPort int, sshServerCircuit *sshd.Circuit) int {
```

Comment:
is there a reason we're renaming this from `builder.Run` to `builder.RunBuilder`? Not too sure how this relates to healthchecks